### DISCUSSION TITLE
`jobs.<job_id>.steps.args` incorrectly documented

### DISCUSSION THREAD

**Discussion by qubyte** [community/community#24866] [Apps, API and Webhooks]
<p>The documentation section in question: <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepsargs" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepsargs</a></p>
<p>The actual path it documents is <code>jobs.\&lt;job_id\&gt;.steps.with.args</code></p>

**Comment by qubyte** [comment]
<p>Or perhaps if it is documented correctly. If it is then I’m getting errors like:</p>
<pre><code>Your workflow file was invalid: .github/workflows/release.yml (Line: 20, Col: 7): Unexpected value 'args'
</code></pre>

**Comment by ScottBrenner** [comment]
<p>Thank you for this! Confirming that adding <code>with</code> works for me.</p>

**Comment by qubyte** [comment]
<p>It looks like a number of the fields documented below this one are also affected.</p>

**Comment by maxheld83** [comment]
<p>I too can confirm this.</p>
<p>This, <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepsargs" rel="nofollow noopener">contra the current docs</a>, does not work:</p>
<pre><code>name: Document Conversion

on: push

jobs:
  convert_via_pandoc:
    name: Convert via Pandoc
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v1
      - uses: maxheld83/pandoc@master
        args: "--standalone --output=index.html README.md"
</code></pre>
<p>This does work, though that’s not how it’s documented:</p>
<pre><code>name: Document Conversion

on: push

jobs:
  convert_via_pandoc:
    name: Convert via Pandoc
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v1
      - uses: maxheld83/pandoc@master
        with:
          args: "--standalone --output=index.html README.md"
</code></pre>
<p>Writing to support to get their attention.</p>

**Comment by maxheld83** [comment, marked as answer]
<p>the docs now appear to be fixed: <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepswithargs" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepswithargs</a></p>
